testsuite/features/step_definitions/command_steps.rb

```diff
@@ -668,7 +677,11 @@
 
 Then(/^terminal "([^"]*)" should have got a retail network IP address$/) do |host|
   node = get_target(host)
-  output, return_code = node.run("ip -4 address show eth1")
+  if host == 'ubuntu-minion'
+    output, return_code = node.run("ip -4 address show ens4")
+  else
+    output, return_code = node.run("ip -4 address show eth1")
+  end
```

Comment:
Shorter:
```
  iface = host == 'ubuntu-minion'? 'ens4': 'eth1'
  output, return_code = node.run("ip -4 address show #{iface}")
```
Please also note that rubocop will bark at you if you ise `"`for constant strings (no interpolation). It recommends to use `'` for constant strings.

Reply:
True